### PR TITLE
Add bobablastic

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fix https://github.com/alltheplaces/alltheplaces/issues/9829

### CODE REVIEW

**Comment by CloCkWeRX** [comment]
This might actually be 50 locations. Storefinder is pretty broken

**Review by davidhicks — CHANGES_REQUESTED** [review]
Suggestions for bringing this spider up to the proposed standard set in #9916:

* Specify `days=DAYS_EN`.
* Remove `time_format`.
* Add `"extras": Categories.FAST_FOOD.value` to `item_attributes`.

Have you tried also adding the following to check if more than 50 locations exist:
```
iseadgg_countries_list = ["US"]
search_radius = 100
max_results = 50
```

If you still see only 50 results with this approach, then seemingly there are indeed 50 stores.

**Comment by CloCkWeRX** [comment]
Oops, lost a store in mexico

File: locations/spiders/bobablastic_us.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from locations.storefinders.wp_store_locator import WPStoreLocatorSpider
+
+
+class BobablasticUSSpider(WPStoreLocatorSpider):
+    name = "bobablastic_us"
+    item_attributes = {"brand_wikidata": "Q108499280", "brand": "Bobablastic", "extras": Categories.FAST_FOOD.value}
+    allowed_domains = [
+        "bobablastic.com",
+    ]
+    time_format = "%I:%M %p"
```
Comment (CloCkWeRX) [inline locations/spiders/bobablastic_us.py:12]:
```suggestion
```

**Review by davidhicks — APPROVED** [review]

**Pull request merged**